src/panels/config/devices/device-detail/integration-elements/zwave_js/device-actions.ts

```diff
@@ -99,6 +99,13 @@ export const getZwaveDeviceActions = async (
       ),
       action: async () => {
         if (
+          isNodeFirmwareUpdateInProgress ||
```

Comment:
I guess the idea if we don't have to show it again even if the firmware update is already done?